index.js

```diff
@@ -44,6 +47,9 @@ module.exports = function(opts) {
       return callback();
     }
 
+    // If HTMLBars, output would be prefixed with Ember.HTMLBars.template function
+    compiled = isHTMLBars ? 'export default Ember.HTMLBars.template(' + compiled + ');' : compiled;
```

Comment:
@mattma yeah, I'm thinking this needs to be done differently somehow as evidenced by the test failure... Thoughts?